course-v2/assets/js/mobile_course_drawers.ts

```diff
@@ -1,17 +1,20 @@
+export const MOBILE_COURSE_NAV_DRAWER_ID = "mobile-course-nav"
+export const MOBILE_COURSE_INFO_DRAWER_ID = "course-info-drawer"
```

Comment:
Making these into constants certainly isn't a bad thing, but I don't see them referenced anywhere else. Is it necessary? Why would just the drawer ID be a constant and not the other ID's?

Reply:
We use the`MOBILE_COURSE_NAV_DRAWER_ID` in the external link's click handler.

https://github.com/mitodl/ocw-hugo-themes/blob/f46f5d7d6fdadf303d737162c2f93d95bf34db83/course-v2/assets/js/external_link_modal.ts#L10

> Is it necessary?

Yes, we use it to close the drawer. Otherwise, the drawer captures external clicks and prevents navigation.

> Why would just the drawer ID be a constant and not the other ID's?

That's a valid question. Right now, only the drawer IDs have the potential for use outside of the original JS file. That's why I exported `MOBILE_COURSE_INFO_DRAWER_ID`, even though it's not in use at the moment. It might seem a bit odd to export just one drawer's ID.

The drawer IDs can be used for opening and closing actions. I didn't think the other IDs would have similar reuse possibilities, so I didn't export them.